Automattic/wp-calypso
# Add PlantUML display cart diagram and infrastructure

#### Changes proposed in this Pull Request

* A diagram of the sequence for displaying a shopping cart using WordPress.com APIs
* A diagram of the sequence for setting the shopping cart using WordPress.com APIs

![Display cart](https://raw.githubusercontent.com/Automattic/wp-calypso/97134c3f9fb3140faf97c376543b08ec3455fda9/client/my-sites/checkout/docs/get-cart.svg)

![Set cart](https://raw.githubusercontent.com/Automattic/wp-calypso/97134c3f9fb3140faf97c376543b08ec3455fda9/client/my-sites/checkout/docs/set-cart.svg)

#### Testing instructions



* Ideally, install a [PlantUML plugin](https://plantuml.com/running) in your preferred IDE and view `client/my-sites/checkout/docs/get-cart.puml`
* View `client/my-sites/checkout/README.md` if your IDE supports it and verify the SVGs are embedded
